sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -80,10 +80,10 @@
     "@azure/core-auth": "^1.1.2",
     "@azure/core-lro": "^1.0.0",
     "@azure/core-paging": "^1.1.0",
-    "@azure/core-http": "^1.0.0",
-    "@azure/core-tracing": "1.0.0-preview.7",
+    "@azure/core-http": "^1.1.1",
+    "@azure/core-tracing": "1.0.0-preview.8",
```

Comment:
Note: we are releasing Form Recognizer preview.1 tomorrow. Dependency version update for form recognizer should be in after the release.

Reply:
OK, I will get the all-clear from you before merging this PR then.